compose/setup.py

```diff
@@ -12,7 +12,7 @@
     url="https://github.com/testcontainers/testcontainers-python",
     install_requires=[
         "testcontainers-core",
-        "docker-compose",
+        # "docker-compose",
```

Comment:
Let's add a comment explaining why this is now commented.

Reply:
added commit for this:

commit 2bd20c6a9ab811343beee150a4816711848fb9ea
Author: Dave Ankin <daveankin@gmail.com>
Date:   Mon Nov 13 06:18:08 2023 -0500

    #392: adding comment to explain why testcontainers-compose no longer depends on docker-compose

diff --git a/compose/setup.py b/compose/setup.py
index 9b783f0..a0b0f84 100644
```diff
--- a/compose/setup.py
+++ b/compose/setup.py
@@ -12,6 +12,7 @@ setup(
     url="https://github.com/testcontainers/testcontainers-python",
     install_requires=[
         "testcontainers-core",
+        # docker compose support has been removed to fix dependency issues in the build
         # "docker-compose",
     ],
     python_requires=">=3.7",
```